package.json

```diff
@@ -107,7 +106,6 @@
     "compression-webpack-plugin": "^0.3.2",
     "css-loader": "^0.26.2",
     "cssnano": "^3.10.0",
-    "dotenv": "4.0.0",
```

Comment:
I've noticed that dotenv has been removed. If we are still using it probably worth putting it back in?